pkg/controller/controllerutil/controllerutil_test.go

```diff
@@ -700,6 +700,42 @@ var _ = Describe("Controllerutil", func() {
 			})
 		})
 
+		Describe("AddFinalizer", func() {
```

Comment:
I think you need to change the name, otherwise Ginko de-duplicates, the junit shows two instances of `AddFinalizer` and not four: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_controller-runtime/1636/pull-controller-runtime-test-master/1512120552670629888

Reply:
The new tests' descriptions have been changed to be different than the originals